test/runtime/adapter-config.spec.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+var juttle_test_utils = require('./specs/juttle-test-utils');
+var check_juttle = juttle_test_utils.check_juttle;
+var path = require('path');
+var expect = require('chai').expect;
+var _ = require('underscore');
+var adapters = require('../../lib/runtime/adapters');
+
+
+function configure(name, conf) {
+    conf = _.clone(conf);
+    conf.path = path.resolve(__dirname, '../runtime/test-adapter-conf');
```

Comment:
I thought about how to do this and implemented it. I guess another way would be to use `sinon`. Can you take a look and see if it's an acceptable solution?